src/app/viam-transport.ts

```diff
@@ -1,5 +1,5 @@
 import { grpc } from '@improbable-eng/grpc-web';
-import { dialDirect } from '@viamrobotics/rpc';
+import { dialDirect } from '../rpc/dial';
```

Comment:
What do you think about adding an `rpc/index.ts` to simplify imports?

Reply:
sure!